app-games/src/main/java/com/aptoide/android/aptoidegames/theme/Color.kt

```diff
@@ -2,30 +2,33 @@ package com.aptoide.android.aptoidegames.theme
 
 import androidx.compose.ui.graphics.Color
 
+//AptoideGames colors
+val primary = Color(0xFFC8ED4F)
+val secondary = Color(0xFF913DD8)
+
+val agBlack = Color(0xFF1E1E26)
+val greyDark = Color(0xFF312D35)
+val grey = Color(0xFF595959)
```

Comment:
Since the design of this app is only using the spelling 'grey' and the UI is separate from other apps, I think we can keep the 'grey' one. The 'gray' instances in this app module will eventually be removed

Reply:
Okay so let's keep it like this since the other ones will eventually be removed